test/integration/node-specific/errors.ts

```diff
@@ -0,0 +1,40 @@
+import { expect } from 'chai';
+
+import { MongoClient, MongoError } from '../../mongodb';
+
+describe('Error (Integration)', function () {
```

Comment:
Because the behavior of the test depends on Node version (AggregateError is only defined Node16+), I added these as integration tests instead of unit tests.

Reply:
Our isAggregateError check is not strict enough to need us to test with actual AggregateErrors, so we could make these unit tests, I don't feel very strongly but we just wouldn't need a node filter if we moved these

Reply:
AggregateErrors are currently the only errors that fit this description that we handle - I'd prefer to test with the actual errors we expect to see than creating an error that conforms to our contract for unit testing.